openerp/addons/base/res/res_company_view.xml

```diff
@@ -50,7 +50,7 @@
                         </page>
                         <page name="configuration" string="Configuration">
                             <group>
-                                <group name="account_grp" string="Accounting"/>
```

Comment:
we knew that. that's why we already replated see enterprise branch  :) 

https://github.com/odoo/enterprise/pull/489/files#diff-b785a5b9a1502707682065378001071eR8